core/consensus/babe/impl/babe_impl.hpp

```diff
@@ -108,7 +108,7 @@ namespace kagome::consensus::babe {
      */
     void startNextEpoch();
 
-    BabeLottery::SlotsLeadership getEpochLeadership(
+    void getEpochLeadership(
```

Comment:
I suggest updating the method name too.
The behavior is a bit inconsistent now. It has "get" in its name and returns nothing.